packages/wonder-blocks-data/docs.md

```diff
@@ -1 +1,60 @@
-In progress!
+## initializeCache
+
+Wonder Blocks Data caches data in its response cache. This cache can be
+initialized with data using the `initializeCache` method. The `initializeData`
+method can only be called once to populate the cache. Usually, the data to be
+passed to `initializeCache` will be obtained by calling `fulfillAllDataRequests`
+after tracking data requests (see [TrackData](#trackdata)) during server-side
+rendering.
+
+This method can also be used to initialize the response cache for tests.
+
+### Usage
+
+```js static
+initializeCache(sourceCache: ResponseCache): void;
+```
+
+#### Function arguments
+
+| Argument | Flow&nbsp;Type | Default | Description |
+| --- | --- | --- | --- |
+| `sourceData` | `ResponseCache` | _Required_ | The source cache that will be used to initialize the response cache. |
+
+### Types
+
+#### ResponseCache
+
+```js static
+type HandlerSubcache = {
+    [key: string]: any,
+    ...,
+};
+
+type ResponseCache = {
+    [handlerType: string]: HandlerSubcache,
+    ...,
+};
+```
+
+A response cache is divided into subcaches by handler type. An example is
+shown below.
+
+```js static
+const responseCache = {
+    HANDLER_TYPE_A: {
+        DATA_ID_1: "DATA STRING",
+        DATA_ID_2: ["array", "of", "data"],
+    },
+    HANDLER_TYPE_B: {
```

Comment:
They'll share the same cache and potentially fulfill each other's requests. That's by design. It makes for a simpler declarative usage rather than having to register handlers.

Reply:
To answer your question, nothing bad will happen.

On the clientside, they would share cache; on the serverside, the first one encountered would fulfill requests for any others of a matching type.